src/styles/index.scss

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dashed #000;
+  border-radius: 50%;
+  border-style: dotted;
```

Comment:
The `border-style: dotted;` is redundant because the `border` property already specifies the style as `dashed`. Consider removing this line to avoid confusion.